src/libs/auth.ts

```diff
@@ -55,7 +68,7 @@ export const getOidcConfig = () => {
   };
 };
 
-const getAuthInstance = () => {
+const getAuthInstance = (): AuthContextProps | undefined => {
```

Comment:
Is there ever a case where this is called before `authContext` is initialized. I suspect there's not.

Could we try having this function return `AuthContextProps` and throw an error if `authContext` is not initialized (as a guard rail to alert us if we're using it in a place we shouldn't)?